components/gc-featured-link/index.json-ld

```diff
@@ -7,25 +7,38 @@
 		"modified": "dct:modified"
 	},
 	"title": {
-		"en": "Featured link - Documentation and working example",
-		"fr": "Lien en vedette - Documentation et example pratique"
+		"en": "Featured link",
+		"fr": "Lien en vedette"
 	},
 	"description": {
 		"en": "This adds a featured link to a page that your institution needs to feature prominently.",
 		"fr": "Ceci ajoute un lien en vedette court et descriptif vers une page que votre institution doit mettre en évidence."
 	},
-	"modified": "2021-05-20",
+	"modified": "2024-02-21",
 	"componentName": "gc-featured-link",
-	"status": "provisional",
+	"status": "stable",
+	"version": "1.1",
```

Comment:
I think this is the version 1 for this component, we are not controlling any version prior the component first stabilization.

```suggestion
	"version": "1.0.0",
```